editor/themes/editor_theme_manager.cpp

```diff
@@ -1679,7 +1680,12 @@ void EditorThemeManager::_populate_editor_styles(const Ref<EditorTheme> &p_theme
 		p_theme->set_stylebox("BottomPanel", EditorStringName(EditorStyles), style_bottom_panel);
 
 		// Main menu.
-		p_theme->set_stylebox("MenuHover", EditorStringName(EditorStyles), p_config.button_style_hover);
+		Ref<StyleBoxFlat> menu_transparent_style = p_config.button_style->duplicate();
+		menu_transparent_style->set_bg_color(Color(1, 1, 1, 0));
+		menu_transparent_style->set_border_width_all(0);
+		Ref<StyleBoxFlat> menu_hover_style = p_config.button_style_hover->duplicate();
```

Comment:
Why are you duplicating this if it's the same?

Reply:
I was told that Godot is more verbose intentionally to reduce confusion so I wasn't really sure how far that went, i.e. if we needed to make sure everything used in this scope could easily be edited later with less line changes. I can remove the duplicate in my next push